manager/ui/war/plugins/api-manager/html/consumer/consumer-orgs.html

```diff
@@ -48,14 +48,24 @@ <h1 class="consumer-top-header" apiman-i18n-key="find-org">Find an Organization<
               <div class="col-md-3"></div>
             </div>
           </div>
-
           <div ng-show="orgs.length > 0">
-	        <div class="count">
-	          Found {{ orgs.length }} matching organizations.
-	        </div>
+            <div class="row">
+	        <div class="count col-md-8">
+	          Found {{resultCount}} matching organizations.
+          </div>
+          <div class="col-md-3 left-spacer-30" ng-show="resultCount > 10">
+            <span apiman-i18n-key="select-page-size">Select page size </span>
+            <select ng-model="pageSize" ng-init="pageSize = '10'" ng-change="searchOrg(orgName, currentPage, pageSize)">
+              <option value="10" ng-selected="true">10</option>
+              <option value="25">25</option>
+              <option value="50">50</option>
+            </select>
+          </div>
+
+        </div>
 	        <div class="item" ng-repeat="org in orgs">
 	          <div class="title"><i class="fa fa-shield icon"></i><a href="{{ pluginName }}/browse/orgs/{{ org.id }}">{{ org.name }}</a></div>
-	          <div class="description" title="{{ org.description }}">{{ org.description }}</div>
+	          <div class="description" title="{{ org.description }}">{{ org.description | limitTo: 60 }}<span ng-show="org.description.length > 60">...</span></div>
```

Comment:
I'm fine with this approach.  I just wanted to point out that it may also be possible to accomplish this using CSS (as an alternative approach).  There are `overflow` and `text-overflow` CSS properties for this.  They are a little bit tricky, however, so not sure about this specific use-case.  Not something I was aware of when I originally wrote this page. :)

Reply:
thanks for the hint, I've learned a new thing today! ;)